src/test/java/testsuite/regression/StatementRegressionTest.java

```diff
@@ -12749,4 +12748,53 @@ public void testBugIssue464() throws Exception {
         }
         assertEquals(3, counter);
     }
+
+    /**
+     * Test fix for Issue #484. Verifies that getGeneratedKeys() will work after executing both batch and individual
+     * statements.
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testBugIssue484() throws Exception {
+        Properties props = new Properties();
+        props.setProperty(PropertyKey.sslMode.getKeyName(), SslMode.DISABLED.name());
+        props.setProperty(PropertyKey.allowPublicKeyRetrieval.getKeyName(), "true");
+        this.conn = getConnectionWithProps(props);
+
+        String tableName = "testBugIssue484";
+
+        createTable(tableName, "(c0 INT AUTO_INCREMENT PRIMARY KEY)");
+
+        Statement stmt = conn.createStatement();
+        stmt.addBatch("INSERT INTO " + tableName + " VALUES(1)");
+        stmt.addBatch("INSERT INTO " + tableName + " VALUES(2)");
+
+        try {
+            stmt.executeBatch();
+        } catch (BatchUpdateException e) {
+            fail();
+        }
```

Comment:
Why do we explicitly catch and fail here instead of adding the exception to the method signature?